pet-store-react-hooks/src/Home/petsTable.css

```diff
@@ -0,0 +1,93 @@
+table {
```

Comment:
I want component css files to have the same name as the component that imports them - capitalize the first letter.